src/system/AdminModule/Controller/AdminInterfaceController.php

```diff
@@ -0,0 +1,773 @@
+                );
+            }
+        }
+
+        foreach ($adminlinks as &$item) {
+            usort($item, 'Zikula\AdminModule\Controller\AdminController::_sortAdminModsByOrder');
+        }
+
+        $menuoptions = array();
+        $possible_cids = array();
+        $permission = false;
+
+        if (isset($categories) && is_array($categories)) {
+            foreach ($categories as $category) {
+                // only categories containing modules where the current user has permissions will
+                // be shown, all others will be hidden
+                // admin will see all categories
+                if ( (isset($adminlinks[$category['cid']]) && count($adminlinks[$category['cid']]) )
+                        || SecurityUtil::checkPermission('.*', '.*', ACCESS_ADMIN) ) {
+                    $menuoption = array(
+                        'url' => $this->get('router')->generate('zikulaadminmodule_admin_adminpanel', array('acid' => $category['cid'])),
+                        'title' => $category['name'],
+                        'description' => $category['description'],
+                        'cid' => $category['cid']);
+                    if (isset($adminlinks[$category['cid']])) {
+                        $menuoption['items'] = $adminlinks[$category['cid']];
+                    } else {
+                        $menuoption['items'] = array();
+                    }
+                    $menuoptions[$category['cid']] = $menuoption;
+                    $possible_cids[] = $category['cid'];
+
+                    if ($acid == $category['cid']) {
+                        $permission =true;
+                    }
+                }
+            }
+        }
+
+        // if permission is false we are not allowed to see this category because its
+        // empty and we are not admin
+        if ($permission==false) {
+            // show the first category
+            $acid = !empty($possible_cids) ? (int)$possible_cids[0] : null;
+        }
+
+        return ['currentcat' => $acid,
+        		'menuoptions' => $menuoptions,
+        		'caller' => $caller
+        ];
+    }
+    
+    
+    /**
+     * @Route("/notices")
+     *
+     * @Template()
+     * @Theme("admin")
+     *
+     * Open the admin container
+     *
+     * @return Response symfony response object
+     */
+    public function noticesAction()
+    {
+    	
+    	if (!SecurityUtil::checkPermission('ZikulaAdminModule::', '::', ACCESS_ADMIN)) {
+    		throw new AccessDeniedException();
+    	}
+    	
+    	// security analyzer and update checker warnings
+    	$notices = array();
+    	$notices['security'] = $this->_securityanalyzer();
+    	$notices['update'] = $this->_updatecheck();
+    	$notices['developer'] = $this->_developernotices();
+    	return ['notices' => $notices];
+    }   
+
+    /**
+     * @Route("/header")
+     *
+     * @Template()
+     * @Theme("admin")
```

Comment:
because this method is called only from within a template that is already an "admin" template. Is the Theme annotation needed? I'm not sure.